### DISCUSSION TITLE
Banned from discord?

### DISCUSSION THREAD

**Discussion by detrol2k** [chromiecraft/chromiecraft#7229] [General]
Hey, 

My handle on Discord is detrol2k and today I decided to post on the discord channel one of my fixes that replaces Vulkan with Directx and in alot of cases with 2014+ hardware it can be a drastic improvement. 

I think the mods thought I was a bot and banned me, I can't get into contact through discord to explain the situation too.

And yes, the API has been tested by me on over 30+ machines and 50+ clients and NEVER had any issues, just improvements. Screw directx.

**Comment by BeefSteaker** [comment]
Hi, I've unbanned you. 
In the future I'd recommend being careful posting direct download links, especially as your first and only interaction with the server

**Reply by detrol2k** [reply]
Yeah, I should have taken a private approach first. Thank you!
